db/migrate/20240111193049_create_comments.rb

```diff
@@ -4,8 +4,8 @@ def change
       t.references :user, null: false, foreign_key: true
       t.references :post, null: false, foreign_key: true
       t.text :text
-      t.datetime :updated_at
-      t.datetime :created_at
+      t.datetime :updatedAt
+      t.datetime :createdAt
```

Comment:
- [ ] **[OPTIONAL]** I would suggest you remove these columns. Besides, the fact they do not follow snake case rule for table columns, there is no use for these `updateAt` and `createdAt` columns. The `t.timestamps` takes care of the `updated_at` and `created_at` columns that we need.